packages/tools-iottwinmaker/src/commands/deploy.ts

```diff
@@ -85,7 +93,9 @@ export const handler = async (argv: Arguments<Options>) => {
         workspace role to continue deployment (Y)? Press any other key to abort (n).`,
       });
       if (response.confirmation === 'Y') {
-        await createWorkspaceIfNotExists(workspaceId);
+        // Support gamma endpoint for TwinMaker
+        const isGamma = !!tmEndpoint && tmEndpoint.includes('gamma');
```

Comment:
we should probably remove references to "gamma" since those are internal details

i.e. instead of exposing the concept of gamma, have a way for consumers to override the places that would change if pointing to a specific endpoint